### ISSUE TITLE
Fix CFG price history on portfolio tab

### ISSUE THREAD

**Issue by hieronx** [centrifuge/apps#2380]
Looks like the hosted uniswap v3 subgraph is removed. There is https://thegraph.com/explorer/subgraphs/5zvR82QoaXYFyDEKLZ9t6v9adgnptxYpKpSbxtgVENFV?view=Query&chain=arbitrum-one but that requires an API key. We need to figure out what is the best new data source.

**Comment by 0x4Graham** [comment]
![Image](https://github.com/user-attachments/assets/cdb4e652-c2e5-4eb1-aa65-4921cb0cba63)

Just wondering, is it meant to look like this?

**Comment by sophialittlejohn** [comment]
In this bug ticket only the CFG price graph that you can see if you click to send DEVEL or CFG was fixed.

![Image](https://github.com/user-attachments/assets/4c2bc60b-2402-4fc9-aa88-a89eeb9596ff)

But I looked into why the graph you screenshotted is not showing. Turns out we manually disabled it about 3 months ago, I don't know why (maybe because it really doesn't look good in our app). After that we had a ticket to save space for all graphs on the page while they load to prevent jumping on the screen. 

![Image](https://github.com/user-attachments/assets/1a469c9e-560d-4dfb-a4c7-9a3a0d12dc6c)


So to summarize:
Problem 1: graph was disabled, should we re-enable it?
Problem 2: if the graph is disabled we should NOT save space for the graph (probs best for another bug ticket)

**Comment by sonam-jo** [comment]
@hieronx do you have context on why the graph was disabled? This page will be redesigned as well, so I am fine to keep it disabled and remove the space but defer to you

**Comment by 0x4Graham** [comment]
What was fixed on this ticket?  @sophialittlejohn I don't see any graph or any button that allows me to send?

**Comment by kattylucy** [comment]
This is not something you would be able to test on demo unfortunately since we don't have subquery data. Turns out the graph is there but the subquery is down / no data it wont show.
